step-definitions/sortingPage.js

```diff
@@ -0,0 +1,17 @@
+import { Given, When, Then } from '@wdio/cucumber-framework';
+
+import LoginPage from '../support/pages/login.page.js'; 
+import homePage from '../support/pages/home.page.js';
```

Comment:
Following the naming convention, you have two classes but one starts with a capital letter and one with a lower letter.